.gitattributes

```
@@ -5,9 +5,5 @@
 packages/**/*.d.ts -linguist-vendored
 # These icons are imported from Google.
 # Kept so that we can review the impact when changing the generation script.
-packages/material-ui-icons/lib linguist-generated
-packages/material-ui-icons/material-icons linguist-vendored
-# bundling fixtures
-test/fixtures/bundling/node-esm/index.js linguist-generated
-test/fixtures/bundling/next-webpack4/pages/index.js linguist-generated
-test/fixtures/bundling/packages.js linguist-generated
+packages/material-ui-icons/lib/** linguist-generated
+packages/material-ui-icons/legacy/** linguist-vendored
```

does 
```suggestion
packages/material-ui-icons/legacy/ linguist-vendored
```
work as well?

---

testing it now

---

it doesn't work

---

hm, thanks for testing. So https://github.com/github/linguist/issues/5439#issuecomment-873368832 is just misleading. It does not work like gitattributes or gitignore. Do we have a comment in the file how one can test the patterns? Would be nice to add a note that this should be tested manually because it doesn't work like gitignore or gitattributes.

---

I believe the answer is in https://www.git-scm.com/docs/gitattributes

> The rules by which the pattern matches paths are the same as in .gitignore files (see gitignore[5]), with a few exceptions:

- negative patterns are forbidden
- patterns that match a directory do not recursively match paths inside that directory (so using the trailing-slash path/ syntax is pointless in an attributes file; use path/** instead)

---

Makes sense. Thanks for clarifying. So it's just about gitattributes. Mentioning gitignore is just misleading. The entrypoint for understanding has to be gitattributes. Not gitignore.